db/schema/patch-007-008.sql

```diff
@@ -0,0 +1,175 @@
+-- Add a lockedAt column for account locking.
+
+ALTER TABLE accounts ADD COLUMN lockedAt BIGINT UNSIGNED DEFAULT NULL;
+
+-- the codes to unlock an account
+CREATE TABLE IF NOT EXISTS accountUnlockCodes (
+  uid BINARY(16) PRIMARY KEY NOT NULL,
+  unlockCode BINARY(16) NOT NULL,
+  FOREIGN KEY (uid)
+      REFERENCES accounts(uid)
+      ON DELETE CASCADE
```

Comment:
Makes sense, I'd rather stay consistent. I feel like I should add a test that ensures unlockCodes are deleted when an [account is deleted](https://github.com/mozilla/fxa-auth-db-server/blob/5932f31331a0cefbfec1d6eed5f94288584528b0/test/backend/db_tests.js#L722). I know a user cannot delete their account if it is in the locked state, but that is enforced on the auth-server and not here.